generators/server/phases/writing/index.js

```diff
@@ -31,6 +31,8 @@ function addMavenSnapshotRepository() {
   this.addMavenRepository('snapshot-repo', 'https://oss.sonatype.org/content/repositories/snapshots');
 }
 
+// TODO Remove this method if we don't need it anymore
+// eslint-disable-next-line
 function addEntandoAuthDependencies() {
```

Comment:
I think we should be able to remove this,
All the rest looks good to me :+1: 

Thanks

Reply:
Done ;)